benchmarks/benchmarks/bench_random.py

```diff
@@ -34,11 +34,11 @@ class Randint(Benchmark):
 
     def time_randint_fast(self):
         """Compare to uint32 below"""
-        np.random.randint(0, 2**30, size=10**5)
+        np.random.randint(low=0, high=2**30, size=10**5)
```

Comment:
The API is not dated since I am going to guess that most existing code calls this function as `np.random.randint(somenumber)` and `np.random.randint(somenumber, someothernumber)`. Removing that calling convention from so many of the tests is just asking for trouble later down the line.